packages/html/tsconfig.json

```diff
@@ -5,6 +5,12 @@
     "esModuleInterop": true,
     "outDir": "./dist",
     "rootDir": "src/",
-    "sourceMap": true
+    "sourceMap": true,
+    // This package will only be used server-side on Node 14+, so target the
+    // newest version of the ES spec that Node 14 supports.
+    "target": "ES2020",
```

Comment:
This ensures that `async/`await` syntax isn't transpiled by the TypeScript compiler.